modules/homa/src/lib.rs

```diff
@@ -146,8 +146,8 @@ pub mod module {
 		/// Block number provider for the relaychain.
 		type RelayChainBlockNumber: BlockNumberProvider<BlockNumber = Self::BlockNumber>;
 
-		/// The HomaXcm to manage the staking of sub-account on relaychain.
-		type HomaXcm: HomaSubAccountXcm<Self::AccountId, Balance>;
+		/// The RelaychainInterface to manage the staking of sub-account on relaychain.
+		type RelaychainInterface: HomaSubAccountXcm<Self::AccountId, Balance>;
```

Comment:
The naming `RelaychainInterface` is confusing. Better to make it related to `XCM`.

Reply:
Yes. Renamed it to XcmInterface instead.